src/Plugin/QuickForm/SaatenUnionSpraying.php

```diff
@@ -0,0 +1,427 @@
+<?php
+
+namespace Drupal\farm_saaten_union\Plugin\QuickForm;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\farm_quick\Plugin\QuickForm\QuickFormBase;
+use Drupal\farm_quick\Traits\QuickLogTrait;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Saaten Union Spraying Quick Form.
+ *
+ * @QuickForm(
+ *   id = "saaten_union_spraying",
+ *   label = @Translation("Spraying"),
+ *   description = @Translation("Create a Saaten Union Spraying log."),
+ *   helpText = @Translation("Use this form to record spraying records."),
+ *   permissions = {
+ *     "create input log",
+ *   }
+ * )
+ */
+class SaatenUnionSpraying extends QuickFormBase {
+
+  use QuickLogTrait;
+
+  /**
+   * The entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The logger service.
+   *
+   * @var \Psr\Log\LoggerInterface
+   */
+  protected $logger;
+  
+  /**
+   * Constructs a QuickFormBase object.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Messenger\MessengerInterface $messenger
+   *   The messenger service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager service.
+   * @param \Psr\Log\LoggerInterface $logger
+   *   The logger service.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, MessengerInterface $messenger, EntityTypeManagerInterface $entity_type_manager, LoggerInterface $logger) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition, $messenger);
+    $this->entityTypeManager = $entity_type_manager;
+    $this->logger = $logger;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('messenger'),
+      $container->get('entity_type.manager'),
+      $container->get('logger'),
+    );
+  }
+
+/**
+ * Retrieves a sorted list of active, non-admin user labels.
+ *
+ * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+ *   The entity type manager.
+ *
+ * @return array
+ *   An array of user labels indexed by user ID and sorted alphabetically.
+ */
+  protected function getUserOptions(EntityTypeManagerInterface $entityTypeManager): array {
```

Comment:
`EntityTypeManagerInterface` does not need to be an argument to this function. Use `$this->$entityTypeManger` instead.

Reply:
I made made this update, I removed the argument.